test/fw/ptl/utils/pbs_testsuite.py

```diff
@@ -1476,42 +1475,34 @@ def revert_mom(self, mom, force=False):
             if 'clienthost' in self.conf:
                 conf.update({'$clienthost': self.conf['clienthost']})
             mom.apply_config(conf=conf, hup=False, restart=False)
-            if mom.is_cpuset_mom():
-                self.server.manager(MGR_CMD_CREATE, NODE, None, mom.shortname)
-                # In order to avoid intermingling CF/HK/PY file copies from the
-                # create node and those caused by the following call, wait
-                # until the dialogue between MoM and the server is complete
-                time.sleep(4)
-                just_before_enable_cgroup_cset = time.time()
-                mom.enable_cgroup_cset()
-                # a high max_attempts is needed to tolerate delay receiving
-                # hook-related files, due to temporary network interruptions
-                mom.log_match('pbs_cgroups.CF;copy hook-related '
-                              'file request received', max_attempts=120,
-                              starttime=just_before_enable_cgroup_cset-1,
-                              interval=1)
-                # Make sure that the MoM will generate per-NUMA node vnodes
-                # when the natural node is created below
-                # HUP may not be enough if exechost_startup is delayed
-                restart = True
-                enabled_cpuset = True
         if restart:
             mom.restart()
         else:
             mom.signal('-HUP')
         if not mom.isUp():
             self.logger.error('mom ' + mom.shortname + ' is down after revert')
         a = {'state': 'free'}
-        if enabled_cpuset:
-            # Checking whether the CF file was copied really belongs in code
-            # that changes the config file -- i.e. after enable_cgroup_cset
-            # called above. We're not sure it is always called here,
-            # since that call is in an if
+        self.server.manager(MGR_CMD_CREATE, NODE, None, mom.shortname)
+        if mom.is_cpuset_mom():
```

Comment:
I don't think it's correct to move the enable pbs_cgroups hook code snippet here. It's meant to be done only when revert_to_default option is set in revert_mom().

Reply:
One suggestion is to bring back 'enable_cpuset' flag, set it True under if *revert_to_default, then later, instead of if mom.is_cpuset(), check for if 'enable_cpuset'.